python/jmxquery/__init__.py

```diff
@@ -1,46 +1,56 @@
 #!/usr/bin/env python3
 
 """
-    Python interface to JMX. Uses local jar to pass commands to JMX and read JSON
-    results returned.
+    Python interface to JMX. Uses local jar to pass commands to JMX.
+    Results returned.
 """
 
-import subprocess
+import subprocess32
```

Comment:
I decided to use the subprocess32 module since the python 2.7 version of the subprocess module doesn't include the run() method or the timeout option.  

I should be able use the Popen() method since it is available on both the 2.7 and 3.x versions of python, but it also does not include a timeout option in the 2.7 version of the subprocess module.

Reply:
I will review the differences between the two versions of the subprocess module further to see if I can utilize it instead.

Reply:
Since this code must work with all versions of Python, this import statement needs to be rewritten.  The idiomatic way to write this is:
```
try:
    import subprocess32 as subprocess
except ImportError:
    import subprocess
```
The `as subprocess` syntax allows you to write `subprocess` in the code without worrying whether it's the included subprocess module or the separately installed subprocess32 module.

Reply:
Implemented per your recommendation.